app/controllers/confirmations_controller.rb

```diff
@@ -60,7 +60,11 @@ def show
 
       if @current_user && @current_user.has_admin_rights?
         report_to_gtm({event: "admin_email_confirmed"})
-        redirect_to getting_started_admin_community_path(:id => @current_community.id) and return
+        if feature_enabled?(:onboarding_redesign_v1)
+          redirect_to getting_started_guide_admin_community_path(@current_community)
```

Comment:
Maybe we should use here (in both cases explicit param "id: @current_community.id"
```
redirect_to getting_started_guide_admin_community_path(id: @current_community.id) and return
```
use the same format also in Admin::CommunitiesController